### DISCUSSION TITLE
Question: plugin deletes excess files in directory, is that intended?

### DISCUSSION THREAD

**Discussion by User670** [GriefPrevention/GriefPrevention#563] [Q&A] [answered]
Plugin ver 16.11.6.

I was messing around with messages.yml. I created a "copy of messages.yml" so that I don't mess up the original, but as the plugin was running, that "copy of" was removed from the plugin data directory. Is that intended behavior?

**Comment by RoboMWM** [comment, marked as answer]
I have copies of older configs in my folder and never had it clear out on me.
